doc/src/site/sphinx/tutorials/sw_automl.rst

```diff
@@ -0,0 +1,163 @@
+Train AutoML Model in Sparkling Water
+-------------------------------------
+
+Sparkling Water provides API in Scala and Python for H2O AutoML.
+The following sections describe how to train an AutoML model in Sparkling Water in both languages.
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        First, let's start Sparkling Shell as
+
+        .. code:: shell
+
+            ./bin/sparkling-shell
+
+        Start H2O cluster inside the Spark environment
+
+        .. code:: scala
+
+            import org.apache.spark.h2o._
+            import java.net.URI
+            val hc = H2OContext.getOrCreate(spark)
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: scala
+
+            val frame = new H2OFrame(new URI("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv"))
+            val sparkDF = hc.asDataFrame(frame).withColumn("CAPSULE", $"CAPSULE" cast "string")
+            val Array(trainingDF, testingDF) = sparkDF.randomSplit(Array(0.8, 0.2))
+
+        Create a H2OAutoML instance and configure it according your use case via provided setters. If feature columns are not specified explicitly,
+        all columns excluding label, fold, weight and ignored columns are considered as features.
+
+        .. code:: scala
+
+            import org.apache.spark.ml.h2o.algos.H2OAutoML
+            val automl = new H2OAutoML()
+            automl.setLabelCol("CAPSULE")
+            automl.setIgnoredCols(Array("ID"))
+
+        By default, AutoML goes through a huge space of H2O algorithms and their hyper-parameters which requires some time. If you wish to speed up
+        the training phase, you can exclude some H2O algorithms and limit the number of trained models.
+
+        .. code:: scala
+
+            automl.setExcludeAlgos(Array(ai.h2o.automl.Algo.GLM))
+            automl.setMaxModels(10)
+
+        Train the AutoML model. The training phase returns the best model according to the sortMetric. If it's not specified, the sortMetric is chosen automatically.
+
+        .. code:: scala
+
+            automl.setSortMetric("AUC")
+            val model = automl.fit(trainingDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the estimator as:
```

Comment:
It is available on the model, no longer on the estimator

Reply:
Good catch, thanks!